internal/commands/tunnel/tunnel_status.go

```diff
@@ -70,57 +70,35 @@ func (o *TunnelStatusOptions) Validate() error {
 }
 
 func (o *TunnelStatusOptions) Run(cmd *cobra.Command) error {
-	process, running, err := daemonRunning(daemonContext(cmd.Context()))
-	if err != nil {
-		return err
-	}
-	if !running {
-		pterm.Info.Println("no background tunnel found")
-		return nil
-	}
-
-	logrus.Debugf("tunnel is up(pid: %d)")
-
-	sess, err := utils.GetSession(&utils.SessionConfig{
-		Region:  o.Config.AwsRegion,
-		Profile: o.Config.AwsProfile,
-	})
-	if err != nil {
-		return err
-	}
+	c := exec.Command(
+		"ssh", "-S", "bastion.sock", "-O", "check", "",
+	)
+	out := &bytes.Buffer{}
+	c.Stdout = out
+	c.Stderr = out
 
-	config, err := getTerraformOutput(sess, o.Config.Env)
+	err := c.Run()
 	if err != nil {
-		return err
+		exiterr := err.(*exec.ExitError)
+		status := exiterr.Sys().(syscall.WaitStatus)
+		if status.ExitStatus() != 255 {
+			return err
+		}
+		return fmt.Errorf("can't tunnel status: %s", out)
```

Comment:
I would show the socket information only if it's -l debug, but for users I'd show instead "tunnel is not up". 
Thoughts?